scripts/local-test-build.sh

```
@@ -236,36 +293,36 @@ install_dependencies() {
         print_status "Updating Poetry lock file..."
-        if ! $poetry_bin lock; then
+        if ! "$poetry_bin" lock; then
             print_error "Failed to update lock file"
             exit 1
         fi
         print_success "Lock file updated"
     elif [ "$update_flag" = "--update-deps" ]; then
         if [ -n "$packages" ]; then
             print_status "Updating specific dependencies: $packages"
-            if ! $poetry_bin update $packages; then
+            if ! "$poetry_bin" update $packages; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)